src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -743,22 +743,19 @@ export default abstract class ComparisonStore {
             return Promise.resolve(
                 this.enrichmentAnalysisGroups.result!.reduce(
                     (acc: EnrichmentAnalysisComparisonGroup[], group) => {
-                        let filteredSamples: Sample[] = [];
+                        let filteredSamples: Sample[] = group.samples;
```

Comment:
@kalletlak is it possible this doesn't work when we're in patient mode?  see Tali's ticket. https://github.com/cBioPortal/cbioportal/issues/8383